### PR TITLE
chore: seed nodes and bump versions

### LINKED ISSUE #50: testground/app: create a seed node
a seed node that connect the peers to each other, so the peers don't need to add one another as persistent peers as mandatory anymore

This can help us in the future minimize the amount of inbound/outbound peers per 1 validator for testing bandwidth vs big blocks

### LINKED ISSUE #85: [EPIC] Test-Plan DA Nodes submit PFD and GetSharesByNamespace
## Introduction

We have to refactor out #65 from #77 to a separate test-plan that focuses on these criterias:

- DA Nodes can submit pfds
- Each DA Node can submit large pfds(e.g. 1 LN is submitting 200kb per block) 
- Can all DA Nodes submit a total of 4MB
- GetSharesByNamespace are working for all 3 types
- Does GetSharesByNamespace return back data in less then 15 seconds (block time)? 

### Pre-Requisite
#33 should be complete 

Ref: #84

### LINKED ISSUE #107: testground/tests: wrapper onFailure event
We need to wrap error returns in test-cases with signalling events in order to gracefully shut down tests
Example can be find in lotus soup testkit

### PR DESCRIPTION
**Pull request by Bidon15**
## Overview
We have moved away from persistent peers in favour of address book and seed node creation in celestia-app p2p network

Closes #107
Closes #50
Closes #85 

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: compositions/cluster-k8s/big-blocks/tc-001-val-large-txs/40-vals-180kb-pfd/256mib-0ms.toml [hunk 1]
```diff
@@ -5,19 +5,21 @@
...
-  persistent-peers = "10"
-  submit-times = "10"
-  msg-size = "180000"
+  [global.run]
+    artifact = ""
+    [global.run.test_params]
+      bandwidth = "256Mib"
+      execution-time = "10"
+      latency = "0"
+      msg-size = "160000"
```
Comment (derrandz) [inline compositions/cluster-k8s/big-blocks/tc-001-val-large-txs/40-vals-180kb-pfd/256mib-0ms.toml:18]: Non-Blocking: I wonder if we can fine tune these values to account for padding to result exactly in 180kb.

File: compositions/cluster-k8s/big-blocks/tc-001-val-large-txs/40-vals-180kb-pfd/320mib-200ms.toml [hunk 2]
```diff
@@ -5,19 +5,21 @@
...
-  persistent-peers = "10"
-  submit-times = "10"
-  msg-size = "180000"
+  [global.run]
+    artifact = ""
+    [global.run.test_params]
+      bandwidth = "320Mib"
+      execution-time = "10"
+      latency = "200"
+      msg-size = "180000"
```
Comment (derrandz) [inline compositions/cluster-k8s/big-blocks/tc-001-val-large-txs/40-vals-180kb-pfd/320mib-200ms.toml:18]: Blocking: Should we account for padding here? like in the other file, we used `160kb` ?

File: compositions/cluster-k8s/big-blocks/tc-001-val-large-txs/80-vals-90kb-pfd/320mib-0ms.toml [hunk 3]
```diff
@@ -5,19 +5,21 @@
...
-  persistent-peers = "10"
-  submit-times = "10"
-  msg-size = "90000"
+  [global.run]
+    artifact = ""
+    [global.run.test_params]
+      bandwidth = "320Mib"
+      execution-time = "10"
+      latency = "0"
+      msg-size = "90000"
```
Comment (derrandz) [inline compositions/cluster-k8s/big-blocks/tc-001-val-large-txs/80-vals-90kb-pfd/320mib-0ms.toml:18]: Blocking: Same question

File: compositions/cluster-k8s/big-blocks/tc-001-val-large-txs/80-vals-90kb-pfd/320mib-100ms.toml [hunk 4]
```diff
@@ -5,19 +5,21 @@
...
-  persistent-peers = "10"
-  submit-times = "10"
-  msg-size = "90000"
+  [global.run]
+    artifact = ""
+    [global.run.test_params]
+      bandwidth = "320Mib"
+      execution-time = "10"
+      latency = "100"
+      msg-size = "90000"
```
Comment (derrandz) [inline compositions/cluster-k8s/big-blocks/tc-001-val-large-txs/80-vals-90kb-pfd/320mib-100ms.toml:18]: 👀

File: compositions/cluster-k8s/big-blocks/tc-001-val-large-txs/80-vals-90kb-pfd/320mib-200ms.toml [hunk 5]
```diff
@@ -5,19 +5,21 @@
...
-  persistent-peers = "10"
-  submit-times = "10"
-  msg-size = "90000"
+  [global.run]
+    artifact = ""
+    [global.run.test_params]
+      bandwidth = "320Mib"
+      execution-time = "10"
+      latency = "200"
+      msg-size = "90000"
```
Comment (derrandz) [inline compositions/cluster-k8s/big-blocks/tc-001-val-large-txs/80-vals-90kb-pfd/320mib-200ms.toml:18]: 👁️

File: compositions/cluster-k8s/big-blocks/tc-002-da-sync/100-100-50-1000/320-100mib-0ms.toml [hunk 6]
```diff
@@ -5,20 +5,44 @@
 [global]
   plan = "celestia"
   case = "002-da-sync"
-  total_instances = 2250
+  total_instances = 2255
```
Comment (derrandz) [inline compositions/cluster-k8s/big-blocks/tc-002-da-sync/100-100-50-1000/320-100mib-0ms.toml:8]: Non-Blocking: should the name of the file also reflect the additional 5 seeds? (_applies to the rest of the files_)

File: compositions/cluster-k8s/big-blocks/tc-002-da-sync/100-100-50-1000/320-100mib-100ms.toml [hunk 7]
```diff
@@ -14,12 +14,34 @@
...
+enable_go_build_cache = true
+enabled = true
+go_version = "1.19"
+[groups.build]
+[groups.run]
+[groups.run.test_params]
+latency = "100"
+bandwidth = "320Mib"
+role = "seed"
+
```
Comment (derrandz) [inline compositions/cluster-k8s/big-blocks/tc-002-da-sync/100-100-50-1000/320-100mib-100ms.toml:44]: your editor has failed you

File: compositions/cluster-k8s/big-blocks/tc-002-da-sync/100-100-50-1000/320-100mib-200ms.toml [hunk 8]
```diff
@@ -14,12 +14,34 @@
...
+enable_go_build_cache = true
+enabled = true
+go_version = "1.19"
+[groups.build]
+[groups.run]
+[groups.run.test_params]
+latency = "200"
+bandwidth = "320Mib"
+role = "seed"
+
```
Comment (derrandz) [inline compositions/cluster-k8s/big-blocks/tc-002-da-sync/100-100-50-1000/320-100mib-200ms.toml:44]: Blocking: It feels like a NIT, but I have to insist 😢

**Review by derrandz — CHANGES_REQUESTED** [review]
Looking good!!
Just a small misses here and there.

Also, I did not get a chance to review the following files:
  
  
* ...ositions/cluster-k8s/big-blocks/tc-004-full-light-past/40-40-20-100/320-100mib-100ms.toml  
* ...ositions/cluster-k8s/big-blocks/tc-004-full-light-past/40-40-20-100/320-100mib-200ms.toml  
* ...ositions/cluster-k8s/big-blocks/tc-005-light-das-past/100-100-50-1000/320-100mib-0ms.toml  
* ...itions/cluster-k8s/big-blocks/tc-005-light-das-past/100-100-50-1000/320-100mib-100ms.toml  
* ...itions/cluster-k8s/big-blocks/tc-005-light-das-past/100-100-50-1000/320-100mib-200ms.toml  
* compositions/cluster-k8s/big-blocks/tc-005-light-das-past/40-40-20-100/256-100mib-0ms.toml  
* compositions/cluster-k8s/big-blocks/tc-005-light-das-past/40-40-20-100/320-100mib-100ms.toml  
* compositions/cluster-k8s/big-blocks/tc-005-light-das-past/40-40-20-100/320-100mib-200ms.toml  
* compositions/cluster-k8s/block-reconstruction/32-square-size/1-full-node.toml  
* compositions/cluster-k8s/block-reconstruction/32-square-size/2-full-nodes.toml  
* compositions/cluster-k8s/block-reconstruction/32-square-size/3-full-nodes.toml  
* compositions/cluster-k8s/block-reconstruction/32-square-size/4-full-nodes.toml  
* compositions/cluster-k8s/block-reconstruction/64-square-size/1-full-node.toml  
* compositions/cluster-k8s/block-reconstruction/64-square-size/10-full-nodes.toml  
* compositions/cluster-k8s/block-reconstruction/64-square-size/2-full-nodes.toml  
* compositions/cluster-k8s/block-reconstruction/64-square-size/3-full-nodes.toml  
* compositions/cluster-k8s/block-reconstruction/64-square-size/4-full-nodes.toml
* compositions/cluster-k8s/pfd-1100.toml
* compositions/cluster-k8s/pfd-400.toml  
* compositions/cluster-k8s/sanity/002-da-sync-8.toml  
* compositions/cluster-k8s/sanity/005-light-das-past-8.toml
* compositions/cluster-k8s/sanity/gsbn-8.toml
* compositions/cluster-k8s/sanity/pfd-8.toml


We need to automate the generation and scaffolding of our toml files, it's so head breaking to review.

File: tests/helpers/app-sync/run_validator.go [hunk 9]
```diff
@@ -78,8 +123,29 @@ func RunValidator(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
...
+		if err != nil {
+			return err
+		}
+
+		bt, err := io.ReadAll(met)
+		if err != nil {
+			return err
+		}
+
+		fmt.Println(string(bt))
```
[the remark this answers is not in the document]
Comment (Bidon15) [reply tests/helpers/app-sync/run_validator.go:142]: Thanks for flagging this! It was a debug info that I removed in favour of native-metrics collection

File: compositions/cluster-k8s/big-blocks/tc-002-da-sync/100-100-50-1000/320-100mib-100ms.toml (hunk 7 above)
Comment (Bidon15) [reply compositions/cluster-k8s/big-blocks/tc-002-da-sync/100-100-50-1000/320-100mib-100ms.toml]: done

File: compositions/cluster-k8s/big-blocks/tc-002-da-sync/100-100-50-1000/320-100mib-200ms.toml (hunk 8 above)
Comment (Bidon15) [reply compositions/cluster-k8s/big-blocks/tc-002-da-sync/100-100-50-1000/320-100mib-200ms.toml]: done

File: compositions/cluster-k8s/big-blocks/tc-002-da-sync/100-100-50-1000/320-100mib-0ms.toml (hunk 6 above)
Comment (Bidon15) [reply compositions/cluster-k8s/big-blocks/tc-002-da-sync/100-100-50-1000/320-100mib-0ms.toml:8]: ideally yes. Parking into a separate issue

**Review by derrandz — APPROVED** [review]

File: testkit/appkit/app.go [hunk 10]
```diff
@@ -319,6 +367,8 @@ func AddSeedPeers(path string, peers []string) error {
 		}
 		peersStr.WriteString(fmt.Sprintf("%s:%d%s", peer, port, separator))
 	}
+
+	fmt.Println(peersStr.String())
```
[the remark this answers is not in the document]
Comment (Bidon15) [reply testkit/appkit/app.go]: Removed

File: compositions/cluster-k8s/big-blocks/tc-001-val-large-txs/40-vals-180kb-pfd/320mib-200ms.toml (hunk 2 above)
Comment (Bidon15) [reply compositions/cluster-k8s/big-blocks/tc-001-val-large-txs/40-vals-180kb-pfd/320mib-200ms.toml:18]: Let's do it in another PR from this issue #143

**Pull request merged**